libass/ass_string.c

```diff
@@ -73,3 +77,18 @@ int ass_strncasecmp(const char *s1, const char *s2, size_t n)
     return a - b;
 }
 
+bool ass_sv_iequal_cstr(ASS_StringView s1, const char *s2)
```

Comment:
I'm not 100% sure, but isn't it equivalent to `ass_strncasecmp(s2, s1.str, s1.len)`?

Reply:
No, that would treat the 2 as equal if `s2` _started_ with `s1`.